monadic-eval/paper2/01-intro.tex

```diff
@@ -15,22 +15,22 @@ \section{Introduction}
 The essence of the state-machine based approach was distilled by Van
 Horn and Might in their ``abstracting abstract machines'' (AAM)
 technique, which provides a systematic method for constructing
-abstract interpreters from standard abstract machines like the CEK- or
```

Comment:
This was correct as is.  It's saying CEK-machine or Krivine-machine in shorthand way: CEK- or Krivine-machine.